examples/translate.yaml

```diff
@@ -0,0 +1,222 @@
+# Example: Part of planning and doing a translation - one step, proofreading
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - alice: https://alice.example/
+    translators: https://translators.example/
+    lingo: https://lingovocab.example/#
+
+'@id': rgh:valueflows/valueflows/master/examples/translate.yaml
+'@graph':
+
+  # recipe (fragment: proofreading process)
+
+  - '@id': translators:c404cff5-19c7-453d-b0d8-d8e96055bd0b
+    '@type': RecipeProcess
+    skos:note: Proofread a translated blog post
+
+  - '@id': translators:13cf44e5-eaa1-4ad0-88d5-e2910a73f377
+    '@type': RecipeFlow
+    action: work 
+    classifiedAs: https://www.wikidata.org/wiki/Q834191 # proofreading
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  - '@id': translators:9f112621-3f13-4f48-a3e2-fd2ca99493cd
+    '@type': RecipeFlow
+    action: accept
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q17928402 # blog post
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  - '@id': translators:c7897c39-7f05-4a5d-a487-80e130a2414b
+    '@type': RecipeFlow
+    action: improve 
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q17928402 # blog post
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  # resources
+
+  - '@id': translators:e1721a61-cd47-4556-84b9-8b1b81da15bf
+    '@type': EconomicResource
+    name: Summer of Commoning - A Tour of French Communities and Projects
+    classifiedAs: https://www.wikidata.org/wiki/Q17928402 # blog post
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  # plan
+
+  - '@id': translators:02b39a30-3e04-4305-9656-7f261aa63c84
+    '@type': Process
+    name: Proofread Summer of Commoning
+    hasBeginning: 2018-12-07T08:00:00-5:00
+    hasEnd: 2018-12-09T08:00:00-5:0
+
+  - '@id': translators:a25500e0-0106-43cd-8cbb-e74779488835
+    '@type': Intent
+    skos:note: Proofreading a French to Spanish translation
+    action: work
+    classifiedAs: https://www.wikidata.org/wiki/Q834191 # proofreading
+    receiver: http://www.translators.example/
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+    time:hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 3 # estimate based on 350 words / hour
+    lingo:wordCount:
+      qudt:unit: unit:Words
+      qudt:numericValue: 1025
+    reciprocityQuantity:
```

Comment:
Are you thinking there will be a way to generate this part of the plan from the recipe?

Also, how to give people a way to create formulas for reciprocity and other purposes on all these fields?

Reply:
Oh, just saw your comment in the conversations thread, answer is there.

Reply:
Approach with `scalingFactor` and `factorProperty` doesn't even assume properties from `vf:` namespace. I think implementations may require to use properties with range `qudt:QuantityValue` and `time:Duration`, other than that even meaning of the property shouldn't matter for scaling, `factorVariable` and `factorFormula` should make all the scaling explicit and independent from meaning of the property.

Since you suggest only one quantity, I guess dedicated `reciprocityQuantity` should work for reciprocity requirement. At the same time other quantities can address other requirements.